### ISSUE TITLE
Forward custom properties to img

### ISSUE THREAD

**Issue by AndreasFaust** [rayriffy/urami#2]
Thank you for this handy library! 
It would be great, if custom properties could be forwarded to the `img`-tag. 
Could be done with `$$restProps`:
```svelte
<img
  src={builtSrc}
  srcset={builtSrcset}
  decoding="async"
  loading="lazy"
 {...$$restProps}
  {...{
    alt,
    height,
    width,
    class: klass,
  }}
/>
```

**Comment by rayriffy** [comment]
oh i thought i already did. i will take a look at this on weekends

**Comment by rayriffy** [comment]
published in `svelte-aio@1.0.0`
